src/nidigital/system_tests/test_system_nidigital.py

```diff
@@ -558,6 +558,27 @@ def test_ppmu_source(multi_instrument_session):
     multi_instrument_session.pins['site0/LO0', 'site1/HI0'].ppmu_source()
 
 
+def test_clock_generator_abort(multi_instrument_session):
+    multi_instrument_session.load_pin_map(os.path.join(test_files_base_dir, "pin_map.pinmap"))
+    multi_instrument_session.pins['site0/PinA', 'site1/PinC'].clock_generator_abort()
+
+
+def test_clock_generator_generate_clock(multi_instrument_session):
+    multi_instrument_session.load_pin_map(os.path.join(test_files_base_dir, "pin_map.pinmap"))
+    multi_instrument_session.pins['site0/PinA', 'site1/PinC'].clock_generator_generate_clock(
+        1e6,
+        True)
+
+
+def test_frequency_counter_measure_frequency(multi_instrument_session):
+    measurement_time = datetime.timedelta(milliseconds=5)
```

Comment:
You can get rid of this variable

Reply:
I know.
It was just there to keep the code shorter.